Sync/SyncStateMachine.swift

```diff
@@ -907,13 +945,17 @@ public protocol EngineStateChanges {
 open class Ready: BaseSyncStateWithInfo {
     open override var label: SyncStateLabel { return SyncStateLabel.Ready }
     let collectionKeys: Keys
+    open let enabled: [String]
```

Comment:
What are these for? I don't see them used.

Reply:
We use them here: https://github.com/mozilla-mobile/firefox-ios/pull/3355/files#diff-7a45ce0d70d55bdf297fcacd47e4e874R1124
Because both the scratchpad and the enabled/disabled in engineConfiguration are protected.

Reply:
I think I would rather expose read-only properties on `engineConfigurations` than duplicate the information.